docs/getting-started/edfi-exchange/technology/ed-fi-odsapi-cloud-deployment-for-aws-suite-3.md

```diff
@@ -0,0 +1,26 @@
+# Ed-Fi ODS/API Cloud Deployment for AWS (Suite 3)
+
+![Unicon Logo](https://edfidocs.blob.core.windows.net/$web/img/edfi-exchange/technology/uniconlogo.png)
+
+> [!INFO]
+> The Ed-Fi Alliance no longer supports nor recommends using the "Ed-Fi ODS/API Cloud Deployment for AWS (Suite 3)" offering and is provided here for historical reference.  At the time this offering was developed, direct platform commands were necessary to support deployments out to cloud providers, which this offering relies heavily on.  Based on community feedback, we've migrated our deployments to utilize Docker as a primary deployment strategy, as all major cloud providers, along with on-premise methods to do so.  We recommend referring to [Docker Deployment 2.x](https://edfi.atlassian.net/wiki/spaces/EDFITOOLS/pages/24119348/Docker+Deployment+2.x) on TechDocs and the other Docker offerings here in the Exchange as ways to stand up Ed-Fi on the major cloud providers.
+
```

Comment:
Opportunity to change the above into a Docusaurus admonition and link to the local version of Docker deploy 2.x.